hw/dv/tools/vcs/cover.cfg

```diff
@@ -9,9 +9,13 @@
 -module pins_if     // DV construct.
 -module clk_rst_if  // DV construct.
 -moduletree prim_alert_sender  // prim_alert_sender is verified in FPV.
+-moduletree prim_prince // prim_prince is verified in a separate DV environment.
```

Comment:
Would you please confirm if VCS is okay with this on IPs that don't instantiate these? Example: run UART smoke with cov and look for errors or warnings.

Reply:
Yup - I've collected coverage for several IPs that don't use these sub-blocks and everything seems okay!
Haven't seen any related errors/warning.